src/udp_wrap.h

```diff
@@ -24,13 +24,13 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "async_wrap.h"
 #include "handle_wrap.h"
 #include "uv.h"
 #include "v8.h"
 
 namespace node {
 
+class AsyncWrap;
```

Comment:
`handle_wrap.h` is guaranteed to provide `AsyncWrap` because including it defines a subclass of `AsyncWrap`